components/global/header/ProfileButton.tsx

```diff
@@ -65,28 +68,30 @@ function ProfileButton() {
                 </Button>
               </div>
             </div>
-            <hr className="border-foreground mx-auto w-full opacity-10 md:w-full" />
-            <div className="bg-background-overlay py-space-y flex min-h-[300px] flex-col items-center justify-center rounded-sm">
-              <div className="mb-4">
-                <Icon name="adgangsplatformen" />
-              </div>
-              <SheetDescription className="text-typo-heading-4 text-foreground">
-                Login via Biblotekernes
-              </SheetDescription>
-              <SheetDescription className="text-typo-heading-4 text-foreground mb-4">
-                fælles loginside
-              </SheetDescription>
-              <div>
-                <Button
-                  theme="primary"
-                  onClick={() =>
-                    router.push(`https://dapple-cms.docker/login?current-path=/go-login`)
-                  }
-                  ariaLabel="Log ind med Adgangsplatformen">
-                  LOG IND
-                </Button>
-              </div>
-            </div>
+            {adgangsplatformenLoginUrl && (
+              <>
+                <hr className="border-foreground mx-auto w-full opacity-10 md:w-full" />
+                <div className="bg-background-overlay py-space-y flex min-h-[300px] flex-col items-center justify-center rounded-sm">
```

Comment:
- Remove min-height, height should be fine. 
- Consider a smaller height on mobile. 

I know its a part of a previous PR. Still these things could be nice to fix. 

```suggestion
                <div className="bg-background-overlay py-space-y flex h-[300px] flex-col items-center justify-center rounded-sm">
```

Reply:
Done (on the two boxes)